SRC/about-me/about-me.css

```diff
@@ -0,0 +1,81 @@
+.mainContainer{
```

Comment:
add: 
`body {`
`margin: 0;`
`}`
gets rid of the annoying default margin on body.